### ISSUE TITLE
More "default" recipe groups.

### ISSUE THREAD

**Issue by BasicallyIAmFox** [bluemagic123/tModLoader#3397]
### Do you intend to personally contribute/program this feature?

Yes

### I would like to see this change made to improve my experience with

Mod capability as a Modder

### Description

Add new recipe groups for:
- All pre hardmode and hardmode bars.
- Evil materials.
- Other common recipe groups?

### What does this proposal attempt to solve or improve?

Currently, there many mods that add their own recipe groups for existing vanilla materials (i.e. Shadow Scale/Tissue Sample) since one doesn't exists by default.

**Comment by Chicken-Bones** [comment]
I can see the advantage, but recipe groups also have the disadvantage of only showing the first item in the group as the ingredient, so it's not clear to players when a mod has added a new item to a group

**Comment by BasicallyIAmFox** [comment]
> I can see the advantage, but recipe groups also have the disadvantage of only showing the first item in the group as the ingredient, so it's not clear to players when a mod has added a new item to a group

Yeah, I can see that. But that doesn't means mods can't show items that are in the recipe group.
Maybe tML could show all items that are in the recipe group when "recipe group" material is hovered over?
